pipelines/chunking.pipeline.json

```diff
@@ -22,7 +22,7 @@
       "pfs": {
         "repo": "parse-docs",
         "name": "data",
-        "glob": "/*"
```

Comment:
any reason to switch the chunking to all the docs instead of just the new/changed ones?

Reply:
I ran into an issue previously where running on new/changed documents caused issues. changing `/*` to `/` helped to debug my issues. I can submit a new PR with the original glob setting if that is what you want.

Reply:
I added the /* back